app/src/main/res/layout-land/activity_rescuecode_show.xml

```diff
@@ -7,111 +7,30 @@
     android:id="@+id/rescueCodeShowActivityView"
     tools:context="org.ea.sqrl.activites.create.RescueCodeShowActivity">
 
-    <android.support.constraint.ConstraintLayout
-        android:id="@+id/inputRow1"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="8dp"
-        android:layout_marginStart="8dp"
+    <TextView
+        android:id="@+id/txtRescueCodeShowDescription"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="16dp"
         android:layout_marginTop="16dp"
-        android:focusable="false"
-        android:layoutDirection="ltr"
```

Comment:
Why did we remove these?

I'm not sure if the text was focusable for some reason but that might have been a change to handle accessibility. the layoutDirection is strange though as we have arabic translation that should be able to be rtl :/

Reply:
I think this is only the diff tool screwing up. It's not comparing the same components here.
```
android:layoutDirection="ltr"
android:focusable="false"
```
This code belonges to the `ConstraintLayout` holding the rescue code input and is still present, but was moved over to `rescue_code_input.xml`. There is a `TextView` now in `activity_rescuecode_show.xml` where the `ConstraintLayout` was before, which is of course missing these attributes. So all should be fine. Can you confirm?

Reply:
Correkt